lib/elixir/lib/version.ex

```diff
@@ -111,11 +111,27 @@ defmodule Version do
   end
 
   defmodule InvalidRequirementError do
-    defexception [:message]
+    defexception [:requirement]
+
+    def exception(requirement) do
```

Comment:
Yes, fixed. I actually had them at one point but when I tested it, it gave me weird errors and I thought it's better to be more laxed here, but it was a different unrelated error 😅